src/hardware/openbench-logic-sniffer/api.c

```diff
@@ -497,10 +497,12 @@ static int dev_acquisition_start(const struct sr_dev_inst *sdi)
 			devc->capture_flags & CAPTURE_FLAG_RLE ? "on" : "off",
 			devc->capture_flags & CAPTURE_FLAG_NOISE_FILTER ? "on": "off",
 			devc->capture_flags & CAPTURE_FLAG_DEMUX ? "on" : "off");
+
 	/*
 	 * Enable/disable OLS channel groups in the flag register according
 	 * to the channel mask. 1 means "disable channel".
 	 */
+	devc->capture_flags &= ~0x3c;
```

Comment:
I think this fixes a bug. I neither see where the bits are cleared.

Reply:
OK, I'll make the wording stronger and tell that this fixes a bug.